Forex.md

```diff
@@ -0,0 +1,35 @@
+# A local proxy for Forex rates
+
+Build a local proxy for getting Currency Exchange Rates
+
+__Requirements__
+
+[Forex](forex) is a simple application that acts as a local proxy for getting exchange rates. It's a service that can be consumed by other internal services to get the exchange rate between a set of currencies, so they don't have to care about the specifics of third party providers.
+
+We provide you with an initial scaffold for the application with some dummy interpretations/implementations. For starters we would like you to try and understand the structure of the application, so you can use this as the base to address the following use case:
+
+> An internal user of the application should be able to ask for an exchange rate between 2 given currencies, and get back a rate that is not older than 5 minutes. He should be able to do this at all times, making up to 10.000 requests every day.
```

Comment:
`Users of the application should be able... The application should support 10,000s of requests per day`